guides/bug_report_templates/action_text_gem.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+gemfile(true) do
+  source "https://rubygems.org"
+
+  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+  gem "capybara"
+  gem "rails", "~> 7.1.0"
+  gem "selenium-webdriver", "~> 4.11.0"
+  gem "sqlite3"
+end
+
+require "action_controller/railtie"
+require "active_record/railtie"
+require "active_storage/engine"
+require "action_text/engine"
+
+class TestApp < Rails::Application
+  config.load_defaults Rails::VERSION::STRING.to_f
+
+  config.root = __dir__
+  config.hosts << "example.org"
+  config.eager_load = false
+  config.session_store :cookie_store, key: "cookie_store_key"
+  config.secret_key_base = "secret_key_base"
+
+  config.logger = Logger.new($stdout)
+  Rails.logger  = config.logger
+
+  config.active_storage.service = :local
+  config.active_storage.service_configurations = {
+    local: {
+      root: Dir.tmpdir,
+      service: "Disk"
+    }
+  }
+
+  routes.append do
+    resources :articles, only: [:new, :create, :show]
+  end
+end
+
+ENV["DATABASE_URL"] = "sqlite3::memory:"
+
+Rails.application.initialize!
+
+require ActiveStorage::Engine.root.join("db/migrate/20170806125915_create_active_storage_tables.rb").to_s
+require ActionText::Engine.root.join("db/migrate/20180528164100_create_action_text_tables.rb").to_s
+
+ActiveRecord::Schema.define do
+  CreateActiveStorageTables.new.change
+  CreateActionTextTables.new.change
+
+  create_table :articles, force: true
+end
+
+class Article < ActiveRecord::Base
+  has_rich_text :body
+end
+
+class ArticlesController < ActionController::Base
+  def new
+    @article = Article.new
+
+    render inline: <<~ERB
+      <html>
+        <head>
+          <script type="module">
+            import "https://ga.jspm.io/npm:trix@2.0.7/dist/trix.esm.min.js"
+          </script>
+        </head>
+        <body>
+          <%= form_with model: @article do |form| %>
+            <%= form.rich_text_area :body %>
+            <%= form.button %>
+          <% end %>
+        </body>
+      </html>
```

Comment:
Without the Trix js, the system test won't be able to interact with the `<trix-editor>`.